### PR TITLE
msolve 0.6.8 (new formula)

### PR DESCRIPTION
**Pull request by mahrud**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by mahrud** [comment]
General question: does adding a `livecheck` mean that this bottle will be automatically updated whenever there's a new release on the upstream repository?

**Comment by SMillerDev** [comment]
> General question: does adding a livecheck mean that this bottle will be automatically updated whenever there's a new release on the upstream repository?

No, a livecheck block just allows `brew livecheck` to see if there is a newer version.

File: Formula/m/msolve.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    depends_on "libomp"
+  end
+
+  def install
+    if OS.mac?
+      libomp = Formula["libomp"]
+      ENV["OPENMP_CFLAGS"] = "-Xpreprocessor -fopenmp -I#{libomp.opt_include} #{libomp.opt_lib}/libomp.dylib"
+    end
+
+    system "autoreconf", "-vif"
```
Comment (SMillerDev) [inline Formula/m/msolve.rb:32]: Does it not come with a configure file?

File: Formula/m/msolve.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    system "./configure",
+           "--disable-dependency-tracking",
+           "--disable-silent-rules",
+           "--enable-openmp=yes",
+           "--prefix=#{prefix}",
+           "--libdir=#{lib}",
```
Comment (SMillerDev) [inline Formula/m/msolve.rb:38]:
```suggestion
    system "./configure",
           "--disable-dependency-tracking",
           "--enable-openmp=yes",
           "--libdir=#{lib}",
           *std_configure_args
```

File: Formula/m/msolve.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      x0*x3*x9+x1*x4*x9+x2*x5*x9+x3*x6*x9+x4*x7*x9+x5*x8*x9+x2*x9-3,
+      x0*x4*x9+x1*x5*x9+x2*x6*x9+x3*x7*x9+x4*x8*x9+x3*x9-4,
+      x0*x5*x9+x1*x6*x9+x2*x7*x9+x3*x8*x9+x4*x9-5,
+      x0*x6*x9+x1*x7*x9+x2*x8*x9+x5*x9-6,
+      x0*x7*x9+x1*x8*x9+x6*x9-7,
+      x0*x8*x9+x7*x9-8,
+      x8*x9-9,
+      x0+x1+x2+x3+x4+x5+x6+x7+x8+1
+    EOS
+    system "#{bin}/msolve", "-f", "eco10-31.ms"
```
Comment (SMillerDev) [inline Formula/m/msolve.rb:58]:
```suggestion
    system bin/"msolve", "-f", "eco10-31.ms"
```

**Review by SMillerDev — DISMISSED** [review]

File: Formula/m/msolve.rb (hunk 1 above)
Comment (mahrud) [reply Formula/m/msolve.rb:32]: Nope, just `configure.ac`

**Comment by mahrud** [comment]
> No, a livecheck block just allows `brew livecheck` to see if there is a newer version.

Hmm I see. Okay, I think I will withdraw this pull request for now then, because right now the upstream is releasing new versions frequently (3 in the past couple of months) and it'll be easier to bottle them on an independent tap rather than open a pull request for each one. Once things are more stable I'll reopen this.

**Comment by mahrud** [comment]
I don't have the time to currently, but if you would like to use the formula to open a new PR please feel free to.

**Pull request closed without merging**